### PR TITLE
Fix some issues in MQTT.

### PR DESCRIPTION
**Pull request by 7osCraft**
Fixed listenMqtt not working if stopped and ran again from the same instance. (#801).
Also fixed an error with thread images being sometimes undefined. (#787).

### CODE REVIEW

File: src/listenMqtt.js [hunk 1]
```diff
@@ -502,6 +502,10 @@ module.exports = function (defaultFuncs, api, ctx) {
   return function (callback) {
     globalCallback = callback;
 
+    //Reset some stuff
+    lastSeqId = 0;
```
Comment (Schmavery) [inline src/listenMqtt.js]: Should these also be in ctx?

Comment (7osCraft) [reply src/listenMqtt.js]: They are mqtt specific, So I just left them here and reset them when listenMqtt is called and they worked fine.

Comment (Schmavery) [reply src/listenMqtt.js:506]: I can't imagine the mqtt clients are supposed to share this information though

Comment (Schmavery) [reply src/listenMqtt.js:506]: The use case that we're fixing is people who are logging into multiple accounts at the same time.
In this case, there would be one ctx object per "api" object, so if you call listenMqtt on account2 after calling it on account1, it won't reset the values for account1.

If we store the variables globally at the top level of this file, all instances of the api will need to share the values. We don't want any interaction at all between instances of the api.

Comment (7osCraft) [reply src/listenMqtt.js:506]: Alright. Sorry for misunderstanding 😁
I'll change it as soon as I can.

**Comment by Schmavery** [comment]
Looks good, thanks a lot!

**Pull request merged**